tests/e2e/main.go

```diff
@@ -430,13 +430,13 @@ TEST RESULTS
 		}
 	}
 	if len(passedTests) > 0 {
-		report += fmt.Sprintln("\n\nPASSED TESTS:\n")
```

Comment:
these trailing newlines gave me a warning on the CLI when I tried to regenerate the traces, so I removed them